### PR TITLE
feat: changing honey to show github release details and hiding the is…

### PR DESCRIPTION
**Pull request by kaliabadi**
…sues functionality for now

### CODE REVIEW

File: src/components/GithubRelease.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { connect } from 'react-redux';
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { fetchReleases } from '../actions/githubActions';
+import { isCollapsed } from '../actions/collapsedActions';
+import Markdown from 'react-remarkable';
+import { CollapsibleComponent, CollapsibleHead, CollapsibleContent } from 'react-collapsible-component';
+
+const releaseComponent = (repoName, releaseDescription, className, collapsed) => {
```
Comment (shendriksen) [inline src/components/GithubRelease.jsx]: Could be extracted as a separat component.

File: src/components/GithubRelease.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                <CollapsibleHead>{repoName}</CollapsibleHead>
+                <CollapsibleContent isExpanded={collapsed || false}>
+                    <Markdown key={repoName}>{releaseDescription}</Markdown>
+                </CollapsibleContent>
+            </CollapsibleComponent>
+        </div>
+    </div>
+}
+
+const preReleaseData = projects =>
```
Comment (shendriksen) [inline src/components/GithubRelease.jsx]: It would be nice to split this into its own component.

File: src/components/GithubRelease.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        const className = `pre-${repoDetails.name}`;
+
+        if(preRelease) {
+            return releaseComponent(repoDetails.name, preRelease.description, className, projects[className]);
+        } else {
+            return  <div><CollapsibleHead>No upcoming relases planned for {repoDetails.name}</CollapsibleHead></div>
+        }
+    })
+
+const latestReleaseData = projects =>
```
Comment (shendriksen) [inline src/components/GithubRelease.jsx]: It would be nice to split this into its own component.

File: src/provider/github.js [hunk 4]
```diff
@@ -69,10 +72,40 @@ const openIssuesGetRequest = project => apolloFetch({
...
+      }
+    }
+  }`,
+  variables: {
+    repositoryName: project.repository,
+    owner: project.owner
+  }
+}).then((response) => {
+  if (response.errors)
+    throw response;
```
Comment (shendriksen) [inline src/provider/github.js]: Was this our linting that put it onto the next line?

File: src/actions/collapsedActions.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export const RELEASE_NOTE_COLLAPSED = 'RELEASE_NOTE_COLLAPSED';
+
+export function isCollapsed(className, isCollapsed) {
+  return (dispatch) => {
+    var key = className;
```
Comment (L0wry) [inline src/actions/collapsedActions.js]: vardy var var

File: src/reducers/githubReducer.js [hunk 6]
```diff
@@ -1,14 +1,16 @@
 import initialState from './initialState';
-import { GITHUB_FETCH_TOTAL_ISSUES, GITHUB_FETCH_OPEN_ISSUES } from '../actions/githubActions';
+import { GITHUB_FETCH_TOTAL_ISSUES, GITHUB_FETCH_OPEN_ISSUES, GITHUB_FETCH_RELEASES, RELEASE_NOTE_COLLAPSED} from '../actions/githubActions';
```
Comment (shendriksen) [inline src/reducers/githubReducer.js]: It looks like "RELEASE_NOTE_COLLAPSED" can be removed from this reducer now.

**Comment by shendriksen** [comment]
A couple of refactoring comments but looks good! It would be good to get some tests in this project now that we are going to start using it.

File: src/reducers/githubReducer.js [hunk 7]
```diff
@@ -1,14 +1,16 @@
 import initialState from './initialState';
-import { GITHUB_FETCH_TOTAL_ISSUES, GITHUB_FETCH_OPEN_ISSUES } from '../actions/githubActions';
+import { GITHUB_FETCH_TOTAL_ISSUES, GITHUB_FETCH_OPEN_ISSUES, GITHUB_FETCH_RELEASES } from '../actions/githubActions';
 
 export default function github(state = initialState.github, action) {
   const { payload } = action;
 
   switch (action.type) {
```
Comment (L0wry) [inline src/reducers/githubReducer.js]: personal opinion case statements make me sad, you can do 

```
return {
    GITHUB_FETCH_TOTAL_ISSUES: Object.assign({}, state, { totalIssues: payload }),
    GITHUB_FETCH_OPEN_ISSUES: Object.assign({}, state, { openIssues: payload })
}[action.type]
```

Comment (shendriksen) [reply src/reducers/githubReducer.js]:
```
return {
    GITHUB_FETCH_TOTAL_ISSUES: Object.assign({}, state, { totalIssues: payload }),
    GITHUB_FETCH_OPEN_ISSUES: Object.assign({}, state, { openIssues: payload })
}[action.type] || state;
```

I guess you could do this with the default?

**Pull request merged**